src/main/features/offer/views/response.njk

```diff
@@ -0,0 +1,68 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, submitButton, errorSummary, radioGroup %}
+{% from "date.njk" import dateInput %}
+
+{% set title = t('Money claim') %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <h1 class="heading-large form-hint">{{ t('Defendant’s response') }} </h1>
+      <form novalidate method="post">
+        {{ csrfProtection(csrf) }}
+        <div class="description-block">
+          <h2 class="heading-large"> 
+          {{ t('%s’s offer',{
+              postProcess: 'sprintf',
+              sprintf: [claim.claimData.defendant.name]
+            }) }}</h2>
+        </div>
+        <table class="offer-table">
+          <tr>
+              <td>{{ t('The offer') }}</td>
+              <td>{{ offer.content }}</td>
+            </tr>
+            <tr>
+              <td>{{ t('By when') }}</td>
+              <td>{{ offer.completionDate | date }}</td>
+            </tr>
+        </table>
+
+        <h2 class="heading-medium"> {{ t('Do you accept the offer?') }} </h2>
+
+        {{
+        radioGroup(
+          form = form,
+          labelForScreenReadersOnly = true,
+          name = 'option',
+          options = [
+            {
+              bold: false,
+              label: 'Yes',
+              hint: t('You can make a legal agreement between you and %s.',
+                { postProcess: 'sprintf',
+                sprintf:[claim.claimData.defendant.name]
+                }),
+              value: 'StatementType.ACCEPTATION.value',
```

Comment:
You are passing the `'StatementType.ACCEPTATION.value'` string here, not accessing `value` property of `StatementType.ACCEPTATION`. Same in other places below.